main.jsx

```diff
@@ -79,14 +96,14 @@ function Faq() {
       <Links selected="/faq" />
       <section class="mt-12 text-lg space-y-10">
         <div class="space-y-4">
-          <a href="#who-is-the-wintercg" id="what-is-the-wintercg">
+          <a href="#what-is-the-wintercg" id="what-is-the-wintercg">
```

Comment:
At this point it's fine to change the anchors but at some point in the future we should try to leave them stable.

Reply:
This link wasn't pointing to any existing anchor, the change was to make it point to the right place.